build.sh

```diff
@@ -37,6 +37,13 @@ docker image build ./$IMAGE_DIR --tag $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME --tag
     && docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$TAG_NAME \
     && docker push $GCR_IMAGE_REPO/$IMAGE_DIR:$VERSION
 
+rm -f /home/jenkins/.docker/config.json
+
+if [ $$IMAGE_DIR = "terra-jupyter-gatk" ]; then
+  docker push broadinstitute/$IMAGE_DIR:$VERSION
```

Comment:
Has this worked yet? https://hub.docker.com/r/broadinstitute/terra-jupyter-gatk appears empty

Reply:
Not yet..I haven't run this job yet...the job will only be triggered if there's a new version being published...